src/elements/flip-card/flip-card/flip-card.visual.spec.ts

```diff
@@ -104,6 +105,39 @@ describe(`sbb-flip-card`, () => {
       );
     }
 
+    for (const imageAlignment of ['after', 'below']) {
+      it(
+        `grid`,
+        visualDiffDefault.with(async (setup) => {
+          await setup.withFixture(
+            html` <div
+              style=${styleMap({
+                display: 'grid',
+                gridTemplateRows: 'minmax(320px, 1fr)',
+                gridTemplateColumns: 'repeat(2, 1fr)',
+                gridColumnGap: '1rem',
+                gridRowGap: '1rem',
+              })}
+            >
+              <sbb-flip-card>
+                ${content('Summary', imageAlignment as SbbFlipCardImageAlignment, true)}
+              </sbb-flip-card>
+              <sbb-flip-card>
+                ${content('Summary', imageAlignment as SbbFlipCardImageAlignment, true)}
+              </sbb-flip-card>
+              <sbb-flip-card>
+                ${content('Summary', imageAlignment as SbbFlipCardImageAlignment, true)}
+              </sbb-flip-card>
+              <sbb-flip-card>
+                ${content('Summary', imageAlignment as SbbFlipCardImageAlignment, true)}
+              </sbb-flip-card>
+            </div>`,
+          );
+          await waitForImageReady(setup.snapshotElement.querySelector('sbb-image')!);
```

Comment:
In theory all 4 pictures should be awaited, but we can try how stable it is like that :)